src/typescript/frontend/src/components/pages/launch-emojicoin/memoized-launch/components/launch-or-goto.tsx

```diff
@@ -39,13 +41,19 @@ export const LaunchButtonOrGoToMarketLink = ({
           </Link>
         ) : (
           <Button
-            disabled={invalid || typeof registered === "undefined"}
+            disabled={insufficientBalance || invalid || typeof registered === "undefined"}
             onClick={onWalletButtonClick}
             scale="lg"
             style={{ cursor: invalid ? "not-allowed" : "pointer" }}
             scrambleProps={scrambleProps}
           >
-            {t("Launch Emojicoin")}
+            {t(
+              insufficientBalance
+                ? "Insufficient balance"
+                : invalid
+                  ? "Invalid input"
+                  : "Launch Emojicoin"
+            )}
```

Comment:
I think we settled on not blocking the user from registering a market whenever we think the balance is insufficient.

Reply:
Discussed on call, I think I'll do a pop-up if it's insufficient, rather than disabling it